src/saturn_engine/worker_manager/api/jobs.py

```diff
@@ -93,3 +100,35 @@ def post_fetch_states() -> Json[FetchCursorsStatesResponse]:
             session=session,
         )
     return jsonify(FetchCursorsStatesResponse(cursors=cursors))
+
+
+@bp.route("/_start", methods=("POST",))
+def post_start_job() -> Json[JobsStartResponse]:
+    restart: bool = flask.request.args.get("_restart", "false") == "true"
```

Comment:
nit, but since it's a param, it doesn't need the leading `_`

Reply:
Got confused with the previous comment 😬